src/scripts/emailBreachAlerts.js

```diff
@@ -42,7 +42,7 @@ const checkInId = Sentry.captureCheckIn({
 });
 
 // Only process this many messages before exiting.
-const maxMessages = 1000;
+const maxMessages = process.env.EMAIL_BREACH_ALERT_MAX_MESSAGES;
```

Comment:
IIUC this is now a string and not a number, but not sure if that matters (maybe `subClient.pull()` converts it behind the scenes anyways). 🤷

Reply:
Hm yeah good point, I'll can explicitly convert it.